.github/dependabot.yml

```diff
@@ -0,0 +1,72 @@
+version: 2
+updates:
+  # Enable version updates for npm (main package)
+  - package-ecosystem: "npm"
+    directory: "/main"
+    schedule:
+      interval: "weekly"
+      day: "monday"
+      time: "09:00"
+    open-pull-requests-limit: 10
+    assignees:
+      - "rduffyuk"
```

Comment:
If this PR gets merged to the Kvan7 repo, you'd be assigned to every dependabot update 😅 

There's a few other places this happens too.